lib/rubocop/config_loader.rb

```diff
@@ -58,7 +58,11 @@ def merge(base_hash, derived_hash)
 
       def base_configs(path, inherit_from)
         configs = Array(inherit_from).map do |f|
-          f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          if f.include?('~')
+            f = File.expand_path(f)
+          else
+            f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          end
```

Comment:
This throws an error for `enabled.yml` and other similar files. I think it is because there needs to be some way to specify a specific path. Maybe there is some way to combine `File.expand_path` and `File.join` but I'm not sure what it is.